### ISSUE TITLE
Non-English Languages

### ISSUE THREAD

**Issue by murrty** [murrty/youtube-dl-gui#20]
Coming up soon, an update will come out that needs some help, if anyone is willing to contribute.

I tested a way to change the language of applications (in [this project (SoloMatchmaking)](https://github.com/murrty/SoloMatchmaking)), and it works mostly out of the box if you start from scratch, but I can't say the same will apply here since there's at least 200 controls that need localizations done to them (and more if I implement more utilities).

I uploaded a starting file for translating, (see https://github.com/murrty/youtube-dl-gui/blob/master/lang/English.ini), and it's not gonna be the complete one for the update, but it's definitely a start with all the major strings that need replacing. If you want to translate, feel free to create new language files (in the "lang" folder) and create a pull request.

**Comment by iowbeoibewio** [comment]
I will translate the English.ini to Arabic, couple days to ready

**Comment by bovirus** [comment]
@murrty 

Could be usefukl to change the issue thread to a titke that explain better your request, Ex.

"How to add a new language to the program"

I propose to add these infos "How to translate" in a section" Translation" in home page of your github reposiory.

**Comment by murrty** [comment]
You're right, I'll refactor some of the pinned issues for better legibility.

**Comment by maboroshin** [comment]
If the translated version have an internal string style like ```chkUseSelection```, no new strings are added.
Please add the English.ini changes to your language ini.

Change History of English.ini : https://github.com/murrty/youtube-dl-gui/commits/master/Languages/English.ini

For each change, click on the 7-digit hash string from the history (View commit details)

**Comment by Drook35** [comment]
i would love to help with the Greek language
